.github/workflows/ci-scripts/macos.sh

```diff
@@ -19,4 +19,12 @@ if [ $EXIT_VAL == 0 ] && [ "$CFG" == "Release" ]; then
 		EXIT_VAL=$?
 fi
 
+if [ "$CFG" == "Debug" ]; then
+	if [ $? == 0 ] && [ "$CFG" == "Debug" ]; then
```

Comment:
I'd say delete both these if-checks, unless there's a reason to only run the boost unit tests with the debug executable?

Reply:
I really have no idea if there's a reason, but if you want them deleted, then I'll delete them.

Reply:
For the WML tests, that's only on release builds since they tend to cause failures by timing out when using the debug build.